src/packaging/bin/cassandra-reaper

```diff
@@ -45,15 +45,15 @@ if [ -z "$CLASS_PATH" ]; then
 fi
 
 if [ $# -eq 0 ]; then
-    CONFIG_PATH="/usr/local/etc/cassandra-reaper/cassandra-reaper.yaml"
+    CONFIG_PATH="/usr/local/etc/reaper/cassandra-reaper.yaml"
```

Comment:
This is going to affect users that have non docker based installations of Reaper. I'm a little nervous with changing paths like this as folks might not notice the change and lose their custom settings files.

What do you think the upgrade path would be here for debian/rpm users?

Reply:
@adejanovski Not good.   

Yes, if that is the case, we may want leave it using the standard `cassandra-reaper` for the `/usr/local/etc` root.  This has been changed only for naming consistency with the `/etc/reaper` change.  I am good reversing this back as we aren't trying to change anything for `/usr/local/etc`.  Can you think of any other areas that may be an issue?

Reply:
Any path change that's not happening within the Docker container means that upgrades will be problematic. Sadly, that's pretty much everything that's in this PR 😕 
I don't think we can go through with it without breaking upgrades, I'm sorry.

Reply:
@adejanovski After discussing this change, I agree let's not introduce instability of the existing path name with a trivial name change.  

I will make adjustments in the respective areas of this repository (different PR) as well as the reaper-operator where the name was changed from `cassandra-reaper` to `reaper`.

Closing this one as we don't want this merged.